tests/cypress/integration/actions_projects_models/case_95_move_task_to_project.js

```diff
@@ -79,6 +78,9 @@ context('Move a task to a project.', () => {
         });
 
         it('Move a task from task. Attempt to add a 3D task to a project with a 2D task.', () => {
+            cy.goToTaskList();
+            cy.createAnnotationTask(task.name3d, task.label3d, task.attrName3d, task.attrValue3d, archiveName3d);
```

Comment:
Why have you moved it here from preparation stage? 
Creating an annotation task is not a part of case description, so, in this way it looks exactly like preparation.

Reply:
Yes. But in this case, in the Firefox browser when it is running Cypress, an error occurs in ```cvat_server```, which I described above.

Reply:
Could you explain how moving it from before() to it() affects server? (and why it happens in Firefox)

Reply:
Unfortunately, I have no explanation for this. Perhaps this may be related to the Firefox update. Since this problem has not been observed before. I'll check on the updated version of Cypress. If necessary, I will create an issue in the Cypress repository.

Reply:
I believe it is necessary to understand why it happens first, because now I even can't image how these changes relates with server issues. In other case we can't guarantee that this fix solves the problems, but it makes code wrong from semantic point of view (I mean preparation steps are considered like a part of the test case)

Reply:
I created an issue for cypress: https://github.com/cypress-io/cypress/issues/18325
I will also close this PR since the case 95 test will be temporarily disabled to run in Firefox in a separate PR that I am currently preparing.
Also, the changes for case 14 are not relevant.